docs/farming-&-staking/farming/additional-guides/network-info.md

```diff
@@ -36,7 +36,7 @@ Below is a table summarizing the necessary port configurations for uninterrupted
 |-------|---------------------|---------------------------------------------------------|----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
 | 30333 | TCP/UDP (QUIC)      | Always public                                           | Substrate networking P2P   | <ul><li>[Space Acres](../../../category/space-acres/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul> | Connecting to the rest of the network for exchanging blocks, transactions, PoT gossip, etc. |
 | 30433 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Space Acres](../../../category/space-acres/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul> | Connecting to the rest of the network and retrieving archival history.            |
-| 30533 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Space Acres because the node and the farmer share the Subspace networking stack there. |
+| 30533 | TCP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Space Acres because the node and the farmer share the Subspace networking stack there. |
```

Comment:
```suggestion
| 30533 | TCP     | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Space Acres because the node and the farmer share the Subspace networking stack there. |
```